core/src/main/java/org/apache/struts2/dispatcher/multipart/JakartaMultiPartRequest.java

```diff
@@ -142,26 +142,22 @@ protected void processNormalFormField(FileItem item, String charset) throws Unsu
             }
 
             long size = item.getSize();
-            if (size == 0) {
-                values.add(StringUtils.EMPTY);
-            } else if (size > maxStringLength) {
+            if (size > maxStringLength) {
+                LOG.debug("Form field {} of size {} bytes exceeds limit of {}.", item.getFieldName(), size, maxStringLength);
```

Comment:
Just added this line, no other functional changes here

Reply:
I think logging the field name at the debug level is fine in terms of security?

Reply:
I assume so, yet this is a user controlled value so maybe it needs escaping

Reply:
Not sure a field name can even contain newline characters but anyway SonarCloud is happy now